project/MiMa.scala

```diff
@@ -1229,7 +1229,20 @@ object MiMa extends AutoPlugin {
         
         // #23023 added a new overload with implementation to trait, so old transport implementations compiled against
         // older versions will be missing the method. We accept that incompatibility for now.
-        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.remote.transport.AssociationHandle.disassociate")
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.remote.transport.AssociationHandle.disassociate"),
+
+        // #23228 single leader per cluster team
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ClusterEvent#CurrentClusterState.copy"),
```

Comment:
CurrentClusterState is a case class and not internal, can't see how it would be used by libraries except possibly for tests, do we need to care?